trackintel/preprocessing/positionfixes.py

```diff
@@ -163,6 +162,11 @@ def generate_triplegs(positionfixes, staypoints=None, *args, **kwargs):
     positionfixes : GeoDataFrame
         The positionfixes have to follow the standard definition for positionfixes DataFrames.
 
+    method: str
+        Method to create tripelgs. Can be one of the following: ('between_staypoints', )
+        'between_staypoints': A tripleg is then defined as all positionfixes between two staypoints. This method
+            requires either a column ``staypoint_id`` on the positionfixes or passing staypoints as an input.
+
     staypoints : GeoDataFrame, optional
```

Comment:
description of `staypoints` should be before `method`

Reply:
👍